packages/nimble-components/src/menu-button/tests/menu-button.stories.ts

```diff
@@ -49,27 +47,69 @@ const metadata: Meta<MenuButtonArgs> = {
         }
     },
     argTypes: {
+        label: {
+            name: 'default',
+            description: 'The text content of the button.',
+            table: { category: 'slots' }
+        },
+        icon: {
+            name: 'start',
+            description:
+                'Set `slot="start"` to include an icon before the text content. This icon will be hidden when `content-hidden` is set.',
+            table: { category: 'slots' }
+        },
+        endIcon: {
+            name: 'end',
+            description: 'Set \`slot="end"\` to include an icon after the text content. This icon will be hidden when \`content-hidden\` is set.',
+            table: { category: 'slots' }
+        },
+        menu: {
+            description: 'The `nimble-menu` to be displayed when the button is toggled.',
+            table: { category: 'slots' }
+        },
         appearance: {
```

Comment:
maybe in the global config we should sort alphabetically?
https://storybook.js.org/docs/essentials/controls#sorting-controls